src/olympia/addons/tests/test_admin.py

```diff
@@ -766,40 +767,55 @@ def test_can_add(self):
 
         response = self.client.post(
             self.add_url, {
-                'excluded_regions': '["de", "pt-BR"]',
+                'excluded_regions': '["DE", "BR"]',
                 'addon': addon.id},
             follow=True)
         assert response.status_code == 200
+        # print(response.content)
```

Comment:
Did you mean to leave this `print` statement here?

Reply:
🙈

Reply:
https://github.com/mozilla/addons-server/pull/16027/commits/10c983f80efcb5d76e31acc854eb22cf442051fd